apps/server/src/adapters/WebsocketAdapter.ts

```diff
@@ -49,8 +49,22 @@ export class SocketServer implements IAdapter {
 
     this.wss.on('connection', (ws) => {
       let clientId = getRandomName();
-      this.clientIds.add(clientId);
-      logger.info(LogOrigin.Client, `${this.clientIds.size} Connections with new: ${clientId}`);
+      this.clients.set(clientId, { type: ClientTypes.Unknown, identify: false, redirect: '' });
```

Comment:
nice!

We could keep the key as a unique ID we generate, and add the preferred name as a property of the object.
I wonder if that would make it easier for us to go around the issue of multiple instances sharing a name and allow us to improve this in future by adding more properties